ansible-ipi-install/roles/installer/templates/install-config.j2

```diff
@@ -38,6 +38,15 @@ platform:
 {% if (release_version[0]|int > 4) or ((release_version[0]|int == 4) and (release_version[2]|int > 3)) %}
     provisioningNetworkInterface: {{ masters_prov_nic }}
     provisioningNetworkCIDR: {{ provisioning_subnet }}
+{% if prov_dhcp_range is defined and prov_dhcp_range|length %}
+    provisioningDHCPRange: {{ prov_dhcp_range }}
+{% endif %}
+{% if prov_bootstrap_ip is defined and prov_bootstrap_ip|length %}
+    bootstrapProvisioningIP: {{ prov_bootstrap_ip }}
+{% endif %}
+{% if prov_ip is defined and prov_ip|length %}
+    clusterProvisioningIP: {{ prov_ip }}
+{% endif %}
```

Comment:
you mention in the sample, 4.4+ maybe we need to add that logic in here as well to reflect that?

Reply:
You can likely piggyback off line 38.